docs/testing/e2e-testing-standards.md

```diff
@@ -0,0 +1,259 @@
+In ownCloud, we use Playwright for webUI test automation. We benefit from lower barriers to entry, readability, and usability when test standards are consistent across repositories. For example:
+
+- **Reusability:** Enhances reusability by making it easier to reuse functions, locators, shared steps, and other test code. Finding the necessary components and functionalities will be simple for someone working on a test resulting in reduced duplication and the requirement for rework after the initiation of a code review.
+
+- **Facilitates reviews:** Code can be examined more quickly. Since you can quickly determine what the test code is doing, it eases the mental strain of code review.
+
+- **Faster onboarding:** By following naming conventions, new contributors are onboarded more rapidly and feel comfortable enough to contribute to the codebase.
+
+Here are the test standards and guidelines we adhere to when creating Playwright tests at ownCloud.
+
+## Folder Structure:
+- `tests/:`
+
+    - `e2e/`: Main folder containing all (end-to-end) E2E test-related files.
```

Comment:
we use 2 technologies cucumber (BDD-tests) + playwright (test implementation). Can we separate them in the description?

like: 

structure: 
- e2e/cucumber - cucumber
- - features
- - steps - stepDefinition for ui steps and given steps (used api)
- - ....


- e2e/support - playwright
- - api
- - objects
- - .....